starky/src/permutation_stark.rs

```diff
@@ -0,0 +1,238 @@
+//! An example of generating and verifying a STARK to highlight the use of the
+//! permutation argument with logUp.
```

Comment:
Given that we have this example now, as you suggested, I think it'd make sense to remove the `permutation` from the Fibonacci example, as well as the needless columns (used only for the sake of permutation)